jdbc_utils.py

```diff
@@ -0,0 +1,10 @@
+import re
+def to_jdbc_url(dbi_url):
+    DB_URL_REGEX = re.compile(r'postgresql://(?P<username>[^:]*):?(?P<password>\S*)@(?P<host_port>\S*)$')
+    match = DB_URL_REGEX.match(dbi_url)
+    if match:
+        jdbc_url = 'jdbc:postgresql://{}?user={}'.format(
+            match.group('host_port'), match.group('username'))
+        if match.group('password'):
+            jdbc_url += '&password={}'.format(match.group('password'))
+        return jdbc_url
```

Comment:
Would it make more sense to just include this method in [`webservices/utils.py`](https://github.com/18F/openFEC/blob/develop/webservices/utils.py) instead of introducing a new file?  I'm not 100% against this, but given that we already have a `utils.py` file that seems like a natural place for this to live.

Reply:
I thought of adding it to `webservices/utils.py` as well, but decided not to as this had nothing to do with webservices. I am open to suggestions.

Reply:
Okay, if you had also given it some thought and that was the reasoning, I'm 👍 on it. :-)